applications/asset_tracker/README.rst

```diff
@@ -52,6 +52,36 @@ The `LTE Link Monitor`_ application, implemented as part of `nRF Connect for Des
 
 By default, the asset tracker supports firmware updates through :ref:`lib_aws_fota`.
 
+.. _power_opt:
+
+Power optimization
+==================
+
+The Asset Tracker can run in three power modes that are configured in the Kconfig file of the application.
+These settings are currently only supported on the nRF9160 DK.
+Although these power modes are supported in Thingy:91, the settings cannot be changed at runtime.
+
+.. note::
+   Not all cellular network providers support these modes, and the granted parameters can vary between networks.
+
```

Comment:
Could also state that SIM card subscription also effects the availability of the Cellular IoT power saving features. Network operators can also limit the availability of power saving features for Roaming users